src/api/app/controllers/webui/user_controller.rb

```diff
@@ -59,6 +59,7 @@ def show
     return unless User.current == @displayed_user
     @is_displayed_user = User.current == @displayed_user
     @patchinfos = @displayed_user.involved_patchinfos
+    @password_changable = !@configuration['change_password'] && !CONFIG['proxy_auth_mode'] && !CONFIG[:ldap_mode]
```

Comment:
OK, so I double checked and the problem was that I used `CONFIG[:ldap_mode]` instead of `CONFIG['ldap_mode']` and thus it returned nil. So I mixed things up. Will change it.